openwisp_monitoring/device/schema.py

```diff
@@ -145,6 +145,26 @@
                             "tx_carrier_errors": {"type": "integer"},
                         },
                     },
+                    "addresses": {
+                        "type": "array",
+                        "title": "Addresses",
+                        "uniqueItems": True,
+                        "additionalItems": True,
+                        "items": {
+                            "additionalProperties": True,
+                            "title": "Address",
+                            "type": "object",
+                            "properties": {
+                                "proto": {"type": "string"},
```

Comment:
Possible values for `proto` so far are: `static`, `dhcp`, `dhcpv6`. @nemesisdesign do you think we should validate that proto contains any of these values?

Reply:
netjson has only `static` and `dhcp`, for understanding if it's ipv4 or ipv6 we use `family`. Yes I think it's useful to enforce this as well, we can use `enum` or anything similar

Reply:
So we need to send dhcp from the lua script even if ubus returns dhcpv6, right?

Reply:
Same query :smile:

Reply:
What do you mean with ubus returns dhcpv6 exactly?

I think the answer to the question is yes, proto is `dhcp` or `static`, family is `ipv4` or `ipv6`.

Reply:
@nemesisdesign I meant that `ubus call network.interface dump` returns `dhcp` and `dhcpv6` as possible values in the `proto` field so we cannot just send the value from ubus